docs/install.md

```diff
@@ -40,20 +40,38 @@ To install Compose, do the following:
 
         curl -L https://github.com/docker/compose/releases/download/VERSION_NUM/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
 
-     If you have problems installing with `curl`, you can use `pip` instead: `pip install -U docker-compose`
+     If you have problems installing with `curl`, see alternative install
+     options below.
```

Comment:
Ya, it seems like there is no standard way to do that with markdown?  I guess I have to do something like this https://gohugo.io/extras/crossreferences/ , I'll try it out